ci/.bazelrc

```diff
@@ -0,0 +1,56 @@
+# Envoy specific Bazel build/test options.
```

Comment:
You don't need this file, it is copied for ci from root directory in build_setup.sh